pyproject.toml

```diff
@@ -70,7 +71,7 @@ optional = true
 [tool.poetry.group.format.dependencies]
 black = "^22"
 colorama = "^0.4.6"
-isort = "^5.10"
+isort = "<5.12.0"
```

Comment:
Thinking out loud here: this translates to "any isort version lower than 5.12.0" while the old specification was "any version >= 5.10 and <5.11".
I am not sure how we initially decided on 5.10, but it seems reasonable to keep that lower bound because older version might not support the use we make of isort in our code?

Reply:
Added 5.10 as a lower bound in c3cf723.